### PR TITLE
F617 Paginated search results.

### PR DESCRIPTION
**Pull request by rolflocher**

### CODE REVIEW

File: CranberryQueue/Queue/SegmentedModal/SearchController.swift [hunk 1]
```diff
@@ -69,21 +73,21 @@ class SearchController: UIViewController, UISearchBarDelegate, SegmentedChildDel
...
         
         let task = URLSession.shared.dataTask(with: request) { data, response, error in
-            guard let data0 = data else {
+            guard let data = data else {
                 print(error!)
                 return
             }
             do {
-                let jsonRes = try JSONSerialization.jsonObject(with: data0, options: []) as? [String: Any]
+                let jsonRes = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any]
```
Comment (JohnnyPersichilli) [inline CranberryQueue/Queue/SegmentedModal/SearchController.swift]: good idea to change this name

File: CranberryQueue/Queue/SegmentedModal/SearchController.swift [hunk 2]
```diff
@@ -23,6 +23,8 @@ class SearchController: UIViewController, UISearchBarDelegate, SegmentedChildDel
         }
     }
     
+    var currentSearchTerm: String? = nil
```
Comment (JohnnyPersichilli) [inline CranberryQueue/Queue/SegmentedModal/SearchController.swift]: same thing with moving var

Comment (rolflocher) [reply CranberryQueue/Queue/SegmentedModal/SearchController.swift]: It's in the right place I think. The global "token" var above it has a getter so it looks like a function

Comment (JohnnyPersichilli) [reply CranberryQueue/Queue/SegmentedModal/SearchController.swift]: ur right i missed that sorry

**Pull request merged**